purchase_supplier_approve/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 RGB Consulting - Domantas Sidorenkovas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    supplier_approval = fields.Selection([
```

Comment:
I would prefer field name like, supplier_approve_status, or supplier_status, which easier to understand at first glance.
For selection, I would prefer,
[('pending_approve', 'Pending Approval'), ('approved', 'Approved'), ('not_approved', 'Not approved')]
My point is about readability. As such, when you see this column from database,
`supplier_approve_status = 'approved'`, would be easier to understand than, `supplier_approve = 'apd'`